docs/development.md

```diff
@@ -227,7 +227,30 @@ experimental-features = nix-command flakes
 
 ## Quick Usage Guide
 
-Get a Nix development shell:
+### With `direnv`
+
+The recommended way to develop is to install [`direnv`](https://direnv.net/)
+(usually via `apt` or any other package manager), then `cd` to the repository
+root, and run `direnv allow` in the terminal (only needs to be done the first
+time, or when the [`.envrc`](/.envrc) file changes).
+
+From then on, every time you `cd` in that same folder (or any subfolder), the
+appropriate nix configuration will be automatically loaded in your existing
+shell, and unloaded again when you `cd` out of the folder.
+
+Note that it may take some time (~N minutes) for `direnv` to apply the nix
```

Comment:
We should give an order of magnitude for N.

Reply:
`N` was meant to be an order of magnitude (i.e. single digit N, between 1 and 10). Anyways picked a number now for clarity.